server/service/src/processors/README.md

```diff
@@ -0,0 +1,33 @@
+# Processors
+
```

Comment:
Thats like a pub sub? one sender multiple subscribers? Think one more intro sentence what processors are and example of how they are used would be good.

Reply:
Thanks: 5df2a2f517dbe511db23ba48b78ad99865d86573
Also: f91c7e1f53b76d819b19f25d5a10a28915a58c9c